src/main/java/org/broadinstitute/hellbender/tools/walkers/SplitIntervals.java

```diff
@@ -86,11 +86,14 @@
 
     public static final String MIN_CONTIG_SIZE_LONG_NAME = "min-contig-size";
 
+    public static final String INTERVAL_FILE_PREFIX_FULL_NAME = "prefix";
```

Comment:
Recommend a slightly more descriptive full name for this argument, such as `interval-file-prefix`

Reply:
done